tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+# Cannot do editable installs with flit
```

Comment:
but I don't want to use editable installs, modern tox should support PEP-517 now https://github.com/tox-dev/tox/pull/954

Reply:
Great! I've not used Flit before, so should we just remove these comments or are other changes needed to use it with tox?

Reply:
should be possible to just remove  `skipsdist=True` and `     pip install .`

Reply:
With:
```diff
diff --git a/tox.ini b/tox.ini
index df15ca7..7961738 100644
--- a/tox.ini
+++ b/tox.ini
@@ -1,6 +1,4 @@
 [tox]
-# Cannot do editable installs with flit
-skipsdist=True
 envlist =
     lint
     py{36, 37, 38}
@@ -9,9 +7,6 @@ envlist =
 deps = flake8
 whitelist_externals=bash
 commands =
-    # Cannot do editable installs with flit
-    pip install .
-
     # Unit tests
     # TODO {envpython} -m pytest --cov TODO --cov tests {posargs}
```
I get:
```console
$ tox -e py37
ERROR: No setup.py file found. The expected location is:
  /private/tmp/flake8-implicit-str-concat/setup.py
You can
  1. Create one:
     https://packaging.python.org/tutorials/distributing-packages/#setup-py
  2. Configure tox to avoid running sdist:
     https://tox.readthedocs.io/en/latest/example/general.html#avoiding-expensive-sdist

```

Reply:
Same with:
```diff
diff --git a/tox.ini b/tox.ini
index df15ca7..81cdeaf 100644
--- a/tox.ini
+++ b/tox.ini
@@ -1,6 +1,4 @@
 [tox]
-# Cannot do editable installs with flit
-skipsdist=True
 envlist =
     lint
     py{36, 37, 38}
```

Reply:
https://tox.readthedocs.io/en/latest/example/package.html#flit

Reply:
Thanks, updated in ttps://github.com/keisheiled/flake8-implicit-str-concat/pull/16/commits/a9bf736a0328fdd3a9034c72add5a95b1f112be2.

Example: https://github.com/hugovk/flake8-implicit-str-concat/runs/325970907

Reply:
https://github.com/tox-dev/tox/pull/1467